doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,141 @@
+
+For a technical deep dive on the subject, see our blog post on [Change Data Capture](https://materialize.com/change-data-capture-part-1/).
+
+**Note**: Currently, the Open CDC format is only supported for [Avro Kafka](/sql/create-source/avro-kafka) sources. If you're interested in using it for another source type, let us know in our [Slack workspace](https://materialize.com/s/chat).
+
+## Example Open CDC Schema
+
+You define the Open CDC format in the Avro schema when a source is created.
+
+
+```sql
+  CREATE MATERIALIZED SOURCE name_of_source
+  FROM KAFKA BROKER 'kafka_url:9092' TOPIC 'name_of_kafka_topic'
+  FORMAT AVRO USING SCHEMA '<schema goes here>'
+  ENVELOPE MATERIALIZE
+
+```
+
+The following schema specifies that records will consist of `id` and `price` fields.
+
+```[
+  {
+    "type": "array",
+    "items": {
+      "type": "record",
+      "name": "update",
+      "namespace": "com.materialize.cdc",
+      "fields": [
+        {
+          "name": "data",
+          "type": {
+            "type": "record",
+            "name": "data",
+            "fields": [
+              {
+                "name": "id",
+                "type": "long"
+              },
+              {
+                "name": "price",
+                "type": [
+                  "null",
+                  "int"
+                ]
+              }
+            ]
+          }
+        },
+        {
+          "name": "time",
+          "type": "long"
+        },
+        {
+          "name": "diff",
+          "type": "long"
+        }
+      ]
+    }
+  },
+  {
+    "type": "record",
+    "name": "progress",
+    "namespace": "com.materialize.cdc",
+    "fields": [
+      {
+        "name": "lower",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "upper",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "counts",
+        "type": {
+          "type": "array",
+          "items": {
+            "type": "record",
+            "name": "counts",
+            "fields": [
+              {
+                "name": "time",
+                "type": "long"
+              },
+              {
+                "name": "count",
+                "type": "long"
+              }
+            ]
+          }
+        }
+      }
+    ]
+  }
+  ]
+  ```
+
+The updates for a single record might look like this:
+
+```
+{"array":[{"data":{"id":5,"price":{"int":10}},"time":1424849130111,"diff":1}]}
+{"array":[{"data":{"id":5,"price":{"int":12}},"time":1424849130112,"diff":1}]}
+{"array":[{"data":{"id":5,"price":{"int":12}},"time":1424849130111,"diff":-1}]}
+{"array":[{"data":{"id":5,"price":{"int":10}},"time":1424849130113,"diff":-1}]}
+```
+
+This will be summarized as:
```

Comment:
What is "This"? What is the subject of this sentence?

Reply:
No longer present in revised version.